src/tracer/MARBL_forcing_mod.F90

```diff
@@ -0,0 +1,378 @@
+        day, "ATM_FINE_DUST_FLUX from cpl", "kg/m^2/s")
+    CS%diag_ids%atm_coarse_dust = register_diag_field("ocean_model", "ATM_COARSE_DUST_FLUX_CPL", &
+        CS%diag%axesT1, & ! T=> tracer grid? 1 => no vertical grid
+        day, "ATM_COARSE_DUST_FLUX from cpl", "kg/m^2/s")
+    CS%diag_ids%atm_bc = register_diag_field("ocean_model", "ATM_BLACK_CARBON_FLUX_CPL", &
+        CS%diag%axesT1, & ! T=> tracer grid? 1 => no vertical grid
+        day, "ATM_BLACK_CARBON_FLUX from cpl",  "kg/m^2/s")
+
+    CS%diag_ids%ice_dust = register_diag_field("ocean_model", "SEAICE_DUST_FLUX_CPL", &
+        CS%diag%axesT1, & ! T=> tracer grid? 1 => no vertical grid
+        day, "SEAICE_DUST_FLUX from cpl", "kg/m^2/s")
+    CS%diag_ids%ice_bc = register_diag_field("ocean_model", "SEAICE_BLACK_CARBON_FLUX_CPL", &
+        CS%diag%axesT1, & ! T=> tracer grid? 1 => no vertical grid
+        day, "SEAICE_BLACK_CARBON_FLUX from cpl", "kg/m^2/s")
+
+  end subroutine MARBL_forcing_init
+
+  ! Note: ice fraction and u10_sqr are handled in mom_surface_forcing because of CFCs
+  subroutine convert_driver_fields_to_forcings(atm_fine_dust_flux, atm_coarse_dust_flux, &
+                                               seaice_dust_flux, atm_bc_flux, seaice_bc_flux, &
+                                               nhx_dep, noy_dep, atm_co2_prog, atm_co2_diag, &
+                                               afracr, swnet_afracr, ifrac_n, &
+                                               swpen_ifrac_n, Time, G, US, i0, j0, fluxes, CS)
+
+    real, dimension(:,:),   pointer, intent(in)    :: atm_fine_dust_flux   !< atmosphere fine dust flux from IOB
+                                                                           !! [kg m-2 s-1]
+    real, dimension(:,:),   pointer, intent(in)    :: atm_coarse_dust_flux !< atmosphere coarse dust flux from IOB
+                                                                           !! [kg m-2 s-1]
+    real, dimension(:,:),   pointer, intent(in)    :: seaice_dust_flux     !< sea ice dust flux from IOB [kg m-2 s-1]
+    real, dimension(:,:),   pointer, intent(in)    :: atm_bc_flux          !< atmosphere black carbon flux from IOB
+                                                                           !! [kg m-2 s-1]
+    real, dimension(:,:),   pointer, intent(in)    :: seaice_bc_flux       !< sea ice black carbon flux from IOB
+                                                                           !! [kg m-2 s-1]
+    real, dimension(:,:),   pointer, intent(in)    :: afracr               !< open ocean fraction
+    real, dimension(:,:),   pointer, intent(in)    :: nhx_dep              !< NHx flux from atmosphere [kg m-2 s-1]
+    real, dimension(:,:),   pointer, intent(in)    :: noy_dep              !< NOy flux from atmosphere [kg m-2 s-1]
+    real, dimension(:,:),   pointer, intent(in)    :: atm_co2_prog         !< Prognostic atmospheric CO2 concentration
```

Comment:
Please describe the units of `afracr`, `atm_co2_prog` and then next 4 real variables, as recommended in https://github.com/mom-ocean/MOM6/wiki/Code-style-guide#documentation-in-code.

Reply:
Because I added the units on the next line, this comment is not showing as "outdated", but units were added in 9359ab7